src/language/codegen.yaml

```diff
@@ -87,7 +87,24 @@
                                   type: StatementBlock
                               - finalize_block:
                                   brief: "Statement block to be executed after calling linear solver"
-                                  type: StatementBlock                                  
+                                  type: StatementBlock
+                        - LongitudinalDiffusionBlock:
+                            brief: "Extracts information required for LONGITUDINAL_DIFFUSION for each KINETIC block."
+                            nmodl: "LONGITUDINAL_DIFFUSION_BLOCK"
+                            members:
+                              - name:
+                                  brief: "Name of the derivative block"
```

Comment:
Should this be "derivative block", or something else?

Reply:
ty